css-basics/css-grid_image-board/css/styles.css

```diff
@@ -17,9 +19,14 @@ body {
 .image-board__container {
   display: grid;
   grid-template-columns: 1fr 1fr 1fr;
-  grid-template-rows: 300px 200px 300px;
+  grid-template-rows: 200px 200px 300px;
   gap: 1rem;
   padding: 1rem;
+  grid-template-areas: 
+    "bb cc ee"
+    "aa cc ff"
+    "aa dd ff";
```

Comment:
The order of these confused me at first. The image --1 to --6 runs alphabetically from "aa" to "ff" and here you start with "bb". Is this on purpose?

The layout of this still resembles the grid from the handout so I don't see a big issue to merge. :)